### DISCUSSION TITLE
Unable to start the CircleCI Language Server

### DISCUSSION THREAD

**Discussion by jdcoldsmith** [community/community#49705] [Codespaces]
### Select Topic Area

Bug

### Body

I recently installed the CircleCI VS Code extension. However, I noticed that every time I open my codespace (in the browser or local VS Code) I get the error popup shown below. I have searched online but haven't found any solutions for how to fix this error. Any help would be appreciated!😄

![image](https://user-images.githubusercontent.com/65235172/224340136-1f23d5d8-3845-4482-8417-cf8da6b8f176.png)

**Comment by rauzerfabiano** [comment]
Based on the information provided, it seems you are encountering an error related to the CircleCI VS Code extension. The error popup does not seem to be shown in your question, so I cannot provide a specific solution for that error.

However, there are a few general troubleshooting steps you can take:

    Check if there is an updated version of the extension available and install it.
    Uninstall the extension, restart VS Code or your browser, then reinstall the extension.
    Check if there are any conflicting extensions installed that may be causing the error. Try disabling other extensions one by one to see if the error goes away.
    Check if there are any issues or bug reports related to the CircleCI VS Code extension on the developer's website or Github page.

If none of these steps solve the issue, you may want to reach out to the developer for further assistance or try posting a more detailed question with the error message included.

**Reply by jdcoldsmith** [reply]
Copied from ChatGPT. The error is still happening.
